src/site/content/en/accessible/accessible-responsive-design/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Accessible Responsive Design
+authors:
+  - dgash
+  - megginkearney
+  - rachelandrew
+  - robdodson
+date: 2020-05-28
+description: |
+  We know that it's a good idea to design responsively to provide the best multi-device experience, but responsive design also yields a win for accessibility.
+tags:
+  - blog
```

Comment:
Because the content has been out for a while I might suggest that we skip the blog and just publish it in the web.dev/accessible/ learning path.